build.yaml

```diff
@@ -520,6 +527,8 @@ steps:
    dockerFile: batch/Dockerfile.worker
    contextPath: .
    publishAs: batch-worker
+   dependsOn:
+    - hail_ubuntu_image
```

Comment:
This can be deleted now.

Reply:
I think you commented on an old sha, this is gone